plugin.video.svtplay/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.svtplay"
        name="SVT Play"
-       version="4.0.40"
+       version="4.0.41"
        provider-name="nilzen">
```

Comment:
Are you referring to this one?
http://kodi.wiki/view/addon.xml#.3Cnews.3E

So we should update the changelog and the news tag?